docs/getting-started.md

```diff
@@ -1,8 +1,21 @@
-## [WIP] Getting started with wrangler
+## Getting started with 🤠 Wrangler
 
-In this document, we outline a few ways to get started on developing and publishing on the Cloudflare Workers® platform.
+`wrangler` is a command line tool for building [Cloudflare Workers](https://workers.cloudflare.com/).
```

Comment:
This might need some discussion. I feel like the scope has grown, not sure how to word it.